src/app/vault/ciphers.component.html

```diff
@@ -54,6 +45,13 @@
           <br />
           <small appStopProp>{{ c.subTitle }}</small>
         </td>
+        <td *ngIf="organizations.length > 0 && !organization" class="org-name-badge">
```

Comment:
Can we just leave out a `td`? Won't that cause the option button to occupy this slot?

Reply:
Absolutely, I put it in a `td` with the community PR we have in mind, but it can easily be put into a column when that work is resumed.

Reply:
Sorry for being a bit unclear, my worry is that if we omit the `td` element for some items it will cause things to shuffle around and occupy the wrong slots. Or do we always display this `td`?

Reply:
Ah I understand now! I can put the `ngIf` on the `app-org-badge` itself, then the `td` will always be there.

Reply:
Looked into this a bit, when the `td` for the org badge doesn't render, the one after it (the options dropdown) still flows to the right most side of the screen, and the td before it (clickable cipher name) occupies the empty space.
If we move the check and always render the td, then there will be an empty bit of space where the badge would go in cases where it doesn't show.

I can move the ngIf so that we always render the `td`, but right now with the current state of the vault not having headers, it appears fine to leave how it is.